grafana/provisioning/dashboards/prod/pubsub.json

```diff
@@ -16,15 +16,15 @@
     "gnetId": null,
     "graphTooltip": 0,
     "id": 4,
-    "iteration": 1642172189689,
+    "iteration": 1649366630728,
     "links": [],
     "panels": [
         {
             "aliasColors": {},
             "bars": false,
             "dashLength": 10,
             "dashes": false,
-            "datasource": "CloudWatch",
+            "datasource": "CloudWatch-1",
```

Comment:
We should use only 1 datasource caled "CloudWatch" but need to add to its connection details the appropriate `Namespaces of Custom Metrics` field
<img width="760" alt="image" src="https://user-images.githubusercontent.com/8445610/163870180-0763dfa8-bbea-40d0-9831-2b1bf9746f50.png">

Reply:
not sure where to enter this, I think the namespace is already in the json?

Reply:
Oh yeah have it here: https://github.com/3box/ceramic-stats/blob/dev/grafana/provisioning/datasources/datasource.yaml#L17

So as long as we are passing in the environment variable I think we should be good on this. Will just have to be different for each env (prod, tnet, dev)